examples/ioredis/README.md

```diff
@@ -0,0 +1,55 @@
+# Overview
+
+OpenTelemetry IORedis Instrumentation allows the user to automatically collect trace data and export them to the backend of choice (we can use Zipkin or Jaeger for this example), to give observability to distributed systems.
+
+## Tracing backend setup
+
+### Zipkin
+
+- Setup [Zipkin Tracing](https://zipkin.io/pages/quickstart.html)
+
+- Set EXPORTER variable
+
+  ```sh
+  export EXPORTER=zipkin
```

Comment:
Instead of requiring a user to export an environment variable, use `cross-env` module

Reply:
do you have any script name suggestions? I find `start:jaeger` and `start:zipkin` sound like you are starting jaeger or zipkin respectively. And `start-with-jaeger(or zipkin)-as-backend` seems verbose.

Reply:
IMO you don't really need to have multiple backend options. This is only meant to show how the ioredis module works. I would just pick one to remove and keep the other.

Reply:
Sounds good, will do.

Reply:
PS on vacation this week so commits are a little slow :)

Reply:
no need to work on vacation this can wait :)